apps/system/style/cards_view/cards_view.css

```diff
@@ -122,6 +122,21 @@
   transform-origin: 50% 50%;
 }
 
+#cards-view .card.private .screenshotView:after {
+  width: 100%;
+  height: 100%;
+  content: "";
+  position: absolute;
+  left: 0; top: 0;
+  background-color: rgba(57, 46, 84, 0.9);
+  background-image: url(../icons/pb_icon.png);
+  background-repeat: no-repeat;
+  background-size: 50%;
+  background-position: center center;
+  opacity: 1;
+  transition: opacity 0.4s ease;
```

Comment:
I'm pretty sure we'll never see this transition. We should remove it.

(we can maybe have a transition by making the opacity:0 here and 1 when `#cards-view.active` but I'm not sure if it'll perform)